### PR TITLE
Make simple_navigation support different channel order

### PR DESCRIPTION
**Pull request by emailweixu**
Also fix a bug of pioneer2dx_camera.world of incorrect link name

### CODE REVIEW

File: python/social_bot/envs/simple_navigation.py [hunk 1]
```diff
@@ -11,17 +11,19 @@
...
-import gym
-import gym.spaces
 import logging
 import numpy as np
 import os
 import random
 
+import gym
+import gym.spaces
+import gin
```
Comment (hnyu) [inline python/social_bot/envs/simple_navigation.py]: Should we also update the dependency for 'gin'?

Comment (emailweixu) [reply python/social_bot/envs/simple_navigation.py]: tf_agents already depends on gin

Comment (emailweixu) [reply python/social_bot/envs/simple_navigation.py]: You mean for setup.py?

Comment (hnyu) [reply python/social_bot/envs/simple_navigation.py]: I mean if someone wants to use SocialRobot independently, then the 'gin' dependency is not met.

File: setup.py [hunk 2]
```diff
@@ -11,8 +11,7 @@
 setup(
     name='social_bot',
     version='0.0.1',
-    install_requires=['gym', 'numpy',
-                      'matplotlib'],  # And any other dependencies foo needs
+    install_requires=['gin-config', 'gym', 'numpy', 'matplotlib'],
```
Comment (hnyu) [inline setup.py]: The README has manual installation for these libs. Maybe change it to running setup.py? (also include 'psutil')

Comment (emailweixu) [reply setup.py]: Changed

**Pull request merged**